### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by githubgrasp** [oofem/oofem#60]
I have some questions concerning the documentation:

1) What is the best way of compiling the documentation for oofem quickly if one makes changes in one of the subfolders of doc? What are the checks one should do that the documentation still works.

2) I am not familiar with the tikz format of the figures that are used. Is it important/desirable for the figures to have this format? If not, is there any recommendation/limitations which file type or size they should have?

**Comment by Micket** [comment]
Instead of answering, I'll just ask a few more questions instead:
Should we go for markdown + ReadTheDocs instead of doing "old fashioned" PDF's? This is what all the cool kids are doing these days.

Anyway; no, definitely not a requirement to use tikz. If we were to aim for readthedocs, then SVG images would be preferable.

**Comment by githubgrasp** [comment]
I agree that it would be nice to have web page of the docs of the development version that is build automatically.

Concerning the figures, SVG would be great as I use inkscape for all my figures.

**Comment by bpatzak** [comment]
We discussed this recently. 
There is need for User Manual and ReadTheDocs is the definitely nice way to follow. 
On the other hand, we started to work on Theory Manual and here we probably combine markdown for top level structure to link independent documents (PDFs, LaTex, etc). The reason is that most of documentation in this manual will be taken from papers, that are already in LaTex. We can't imagine to write this manual from scratch, so opted for this bazaar style.
